README.md

```diff
@@ -1103,14 +1103,20 @@ column type.
 #### Convert a column from text to json
 
 Postgres' `alter column` command will not automatically convert a `text`
-column to `json`, but it can still be done with plain SQL.
+column to `json`, but it can still be done with a piece of plain SQL.
 
-```sql
-alter table versions
-alter column object type json
-using object::json;
+```ruby
+def up
+  change_column :versions, :object, 'jsonb USING object::jsonb'
+end
+
+def down
+  change_column :versions, :object, 'text USING object::text'
+end
 ```
 
+You'll have to destroy or migrate all `versions` data before running this migration.
```

Comment:
Are you sure? I tested the `alter table` with postgres 9.4 and it seems to convert correctly:

```sql
create table heyojson ( data text not null );
insert into heyojson (data) values ('{ "foo" : "bar" }');
alter table heyojson alter column data type jsonb using data::jsonb;
select * from heyojson;
-- {"foo": "bar"}
```